README.md

```diff
@@ -132,6 +144,9 @@ and set up the following environment variables in order for this MCP to work:
   * `storage:system:read` - Read System Data from Grail
   * `storage:user.events:read` - Read User events from Grail
   * `storage:user.sessions:read` - Read User sessions from Grail
+  * `davis-copilot:conversations:execute` - execute conversational skill
```

Comment:
This is unfortunately a breaking change for many of our users, as they will need new oauth clients with those scopes.
While we are still in version 0.x, I believe we need to figure out a way to keep the mcp-server working, and only requesting those permissions when they are needed, before we merge this PR. I'll create a ticket for that.